doc/source/tutorial/optimize.rst

```diff
@@ -1557,12 +1560,26 @@ We have a table showing the individual medley times of five students:
  E          46.3           47.8         50.4        37.2
 ==========  ===========  ============  ===========  ===============================
 
-We need to choose best four students for each swimming style.
-This is a typical linear sum assignment problem, we can use :func:`linear_sum_assignment` to solve it.
+We need to choose the best student for each of the four swimming styles.
+This is a typical linear sum assignment problem. We can use :func:`linear_sum_assignment` to solve it.
+
+The linear sum assignment problem is one of the most famous combinatorial optimization problems.
+When we are given a matrix :math:`C` known as a "cost matrix" and we want to match each row to a different column,
+we need to find a combinations of a row and a column which the sum of the corresponding entries is minimized
+(or maximized).
+
+Formally, let X be a boolean matrix where :math:`X[i,j] = 1` iff row i is assigned to column j.
+Then the optimal assignment has cost
+
+.. math::
+
+    \min \sum_i \sum_j C_{i,j} X_{i,j}
```

Comment:
```suggestion
Then the problem is

.. math::

   \min_X \sum_i \sum_j  & C_{i,j} X_{i,j} \\
   \text{s.t.} \quad \quad & \sum_i X_{i,j} = 1 \\ 
   & \sum_j X_{i,j} \leq 1
```
Or that's the best way I can figure out to write it in a way the Sphinx should interpret correctly.